app/Http/Requests/StoreBomRequest.php

```
@@ -0,0 +1,38 @@
+    public function rules(): array
+    {
+        return [
+            'task_id' => 'nullable|exists:tasks,id',
+            'resource_id' => 'nullable|exists:resources,id',
+            'material_name' => 'required|string|max:255',
+            'quantity' => 'required|numeric|min:0',
+            'unit' => 'required|string|max:50',
+            'unit_price' => 'nullable|numeric|min:0',
+            'amount' => 'nullable|numeric|min:0',
+            'additional_details' => 'nullable|string',
+            'source_type' => ['required', Rule::in(SourceType::toArray())],
```

PHPMD (2.15.0)

Avoid using static access to class '\Illuminate\Validation\Rule' in method 'rules'. (undefined)

(StaticAccess)